core/src/main/scala/org/apache/spark/HttpFileServer.scala

```diff
@@ -20,6 +20,7 @@ package org.apache.spark
 import java.io.File
 
 import com.google.common.io.Files
+import org.apache.commons.httpclient.util.URIUtil
```

Comment:
Should we consider explicitly adding that dependency? If it's already done somewhere else then it would be best that we don't re-implement the same thing and potentially introduce our own bugs.